src/modules/Main.ui.js

```diff
@@ -50,7 +50,10 @@ import SendConfirmation from './UI/scenes/SendConfirmation/index'
 import Scan from './UI/scenes/Scan/ScanConnector'
 import ExchangeConnector from '../connectors/scene/CryptoExchangeSceneConnector'
 import WalletList from './UI/scenes/WalletList/WalletListConnector'
-import CreateWallet from './UI/scenes/CreateWallet/CreateWalletConnector.js'
+import CreateWalletName from './UI/scenes/CreateWallet/CreateWalletConnector'
```

Comment:
Do not use default exports. This applies to all imports added in this file. Leave any old code alone. No need to clean up right now, but do make sure new code doesn't use default exports.